src/components/DateRangePickerInput.jsx

```diff
@@ -39,8 +39,8 @@ const propTypes = forbidExtraProps({
   onStartDateShiftTab: PropTypes.func,
   onEndDateTab: PropTypes.func,
   onClearDates: PropTypes.func,
-  onArrowDown: PropTypes.func,
-  onQuestionMark: PropTypes.func,
```

Comment:
this makes it a breaking change, unfortunately

Reply:
(doh) sorry I should have known that.  I can split that out, yes.